samples/net/azure_iot_hub/boards/native_sim.conf

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# General
+CONFIG_LOG_BUFFER_SIZE=2048
+CONFIG_HW_ID_LIBRARY=y
+CONFIG_ASSERT=y
+CONFIG_FPU=y
+
+# Heap and stacks
+CONFIG_INIT_STACKS=y
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=8192
```

Comment:
Do we need 8k of RAM for the system workqueue in this sample?

Reply:
no, removed